packages/broker/src/plugins/operator/MaintainTopologyHelper.ts

```diff
@@ -76,10 +68,10 @@ export class OperatorClient extends EventEmitter<OperatorClientEvents> {
             const sponsorshipCount = (this.sponsorshipCountOfStream.get(streamId) || 0) + 1
             this.sponsorshipCountOfStream.set(streamId, sponsorshipCount)
             if (sponsorshipCount === 1) {
-                this.emit("addStakedStream", streamId, await this.contract.provider.getBlockNumber())
+                this.emit("addStakedStream", [streamId])
             }
+            latestBlock = await this.contract.provider.getBlockNumber()
```

Comment:
I think this line is not needed. We would've have already initiated execution of method `pullStakedStreams` when this update would occur, therefore not affecting anything.

Reply:
indeed!